src/test/java/com/osiris/autoplug/client/tasks/updater/server/TaskServerUpdaterTest.java

```diff
@@ -22,7 +22,7 @@
 class TaskServerUpdaterTest {
 
     @Test
-    void testPaper() throws JLineLinkException, NotLoadedException, YamlReaderException, YamlWriterException, IOException, IllegalKeyException, DuplicateKeyException, IllegalListException, InterruptedException {
+    void testMCServerUpdater() throws JLineLinkException, NotLoadedException, YamlReaderException, YamlWriterException, IOException, IllegalKeyException, DuplicateKeyException, IllegalListException, InterruptedException {
```

Comment:
This method should stay named `testPaper()` and you should add more methods like this for each of the available server software, like: `testPufferfish()`, `testPatina()` etc...

Reply:
Btw also just committed a fix that includes `assertTrue(maMan.manager.getAll().get(0).isSuccess());` check here.

Reply:
I added `testPurpur()`.
Too lazy to add the others because it's too much.